geomesa-web/geomesa-web-core/src/main/scala/org/locationtech/geomesa/web/core/SpringScalatraBootstrap.scala

```diff
@@ -11,16 +11,21 @@ package org.locationtech.geomesa.web.core
 import javax.servlet.ServletContext
 import javax.servlet.http.{HttpServletRequest, HttpServletRequestWrapper, HttpServletResponse}
 
+import com.typesafe.scalalogging.slf4j.Logging
+import org.apache.commons.lang.exception.ExceptionUtils
 import org.locationtech.geomesa.accumulo.data.AccumuloDataStoreFactory
-import org.scalatra.ScalatraServlet
 import org.scalatra.servlet.RichServletContext
+import org.scalatra.{InternalServerError, ScalatraServlet}
 import org.springframework.context.{ApplicationContext, ApplicationContextAware}
 import org.springframework.web.context.ServletContextAware
 
 import scala.beans.BeanProperty
 import scala.collection.JavaConversions._
 
-trait GeoMesaScalatraServlet extends ScalatraServlet {
+trait GeoMesaScalatraServlet extends ScalatraServlet with Logging {
```

Comment:
What are we going to do for Java 8?  Scalatra only supports Java 7:  http://www.scalatra.org/getting-started/installation.html#toc_101

Some of the GeoMesa users have already moved beyond Java 7, which makes these features inaccessible to them.